packages/teams/src/utils/process-invite-users.ts

```diff
@@ -0,0 +1,55 @@
+import { ArcGISRequestError } from "@esri/arcgis-rest-request";
+import { IUser } from "@esri/arcgis-rest-types";
+import { getProp, inviteUsers } from "@esri/hub-common";
+import { IAddOrInviteContext, IAddOrInviteResponse } from "../types";
+
+/**
+ * @private
+ * Process invite of users
```

Comment:
Comments should explain what's going on from the context of the Hub system as a whole. So this is inviting a set of users to a specific group